src/frontend/web_application/src/components/ContactProfile/index.spec.js

```diff
@@ -1,7 +1,9 @@
 // import contactModule from '../index.js';
 
 describe('component contact-card-summary', () => {
-  it('needs to be implemented', () => {});
+  it('needs to be implemented', () => {
```

Comment:
Another way to do this is to use `it.skip(` instead of `it(`, so it will be displayed as not passing in the tests overview at the end of the tests run.